src/main/WorkflowSelection.tsx

```diff
@@ -182,15 +181,26 @@ const WorkflowButton: React.FC<{stateName: string, workflowId: string, workflowD
   );
 }
 
-const WorkflowSelectRadio = withStyles({
-  root: {
-    alignSelf: 'start',
-    color: 'grey',
-    "&$checked": {
-      color: 'grey'
-    }
-  },
-  checked: {}
-})((props) => <Radio color="default" {...props} />);
+const WorkflowSelectRadio: React.FC = () => {
+
+  const style = css({
+    root: {
+      alignSelf: 'start',
+      color: 'grey',
+      "&$checked": {
+        color: 'grey'
+      }
+    },
+  })
+
+  return (
+    <Radio
+      color="default"
+      css={style}
+      checked={false}
+    />
+  )
+}
+
```

Comment:
Something doesn't work here. When running this locally, I cannot select any workflow. At least I don't get any visual feedback and looks like none is selected:

![image](https://github.com/opencast/opencast-editor/assets/7419664/014a3c21-1c50-4597-9978-0b26055f8f42)